docs/docs/Integrations/integrations-mcp.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Integrate Langflow with MCP (Model context protocol)
+slug: /integrations-mcp
+---
+
+Langflow integrates with the [Model Context Protocol (MCP)](https://modelcontextprotocol.io/introduction). This allows you to use your Langflow flows as tools in other applications that support the MCP protocol, or extend Langflow with the [MCP stdio component](/components-tools#mcp-tools-stdio) to access MCP servers.
+
+* Access all of your flows as tools from Claude Desktop
+* Connect a [Datastax Astra DB MCP server](https://github.com/datastax/astra-db-mcp) to Langflow
+
+This guide shows you how to use Langflow as an MCP server with Claude Desktop as the client.
+
+## Access all of your flows as tools from Claude Desktop
+
+Add your Langflow MCP server configuration to Claude, so it can access all of your flows as tools with the MCP protocol.
+
+### Prerequisites
+
+* [Claude desktop](https://claude.ai/download) is installed.
+* [uv is installed](https://docs.astral.sh/uv/getting-started/installation/) to enable you to run uvx commands.
+
+### Add Langflow as an MCP server to Claude
+
+1. Create at least one flow, and note your host. For example, `http://127.0.0.1:7863`.
```

Comment:
has anyone tested this with ds langflow?
if not, I wouldn't block this PR on that but maybe do a fast follow?

Reply:
Not the two tasks in this doc. I've made a ticket and i'll do it